frontend/src/component/project/Project/ProjectInsights/ProjectHealth/FlagCounts.tsx

```diff
@@ -0,0 +1,80 @@
+import { Box, styled, useTheme } from '@mui/material';
+import { Link } from 'react-router-dom';
+import type { FC } from 'react';
+
+const Dot = styled('span', {
+    shouldForwardProp: (prop) => prop !== 'color',
+})<{ color?: string }>(({ theme, color }) => ({
+    height: '15px',
+    width: '15px',
+    borderRadius: '50%',
+    display: 'inline-block',
+    backgroundColor: color,
+}));
+
+const FlagCountsWrapper = styled(Box)(({ theme }) => ({
+    display: 'flex',
+    flexDirection: 'column',
+    gap: theme.spacing(2),
+}));
+
+const FlagsCount = styled('p')(({ theme }) => ({
+    color: theme.palette.text.secondary,
+    marginLeft: theme.spacing(3),
+}));
+
+const StatusWithDot = styled(Box)(({ theme }) => ({
+    display: 'flex',
+    alignItems: 'center',
+    gap: theme.spacing(1),
+}));
+
+export const FlagCounts: FC<{
```

Comment:
this code is extracted from the project insights health widget so that we can compose is inside personal dashboard